### DISCUSSION TITLE
Menubar?

### DISCUSSION THREAD

**Discussion by Barugon** [hecrj/iced#1363] [Q&A] [answered]
Has anyone implemented a menubar? Are there any examples of how this would be done?

**Comment by genusistimelord** [comment, marked as answer]
in iced_aw there is a PR for one but it is not yet finished due to some issues. https://github.com/iced-rs/iced_aw/pull/4

**Reply by Barugon** [reply]
> in iced_aw there is a PR for one but it is not yet finished due to some issues. [iced-rs/iced_aw#4](https://github.com/iced-rs/iced_aw/pull/4)

Thanks. Looks like it should work for some shorter menus. I'll give it a go.
